include/mbgl/map/map_options.hpp

```diff
@@ -88,6 +88,24 @@ class MapOptions final {
      */
     bool crossSourceCollisions() const;
 
+    /**
+     * @brief Specify whether render data for layers, sources and images should be kept between renderStill() calls.
+     * If set to false, all the data is created from scratch for every renderStill() call.
```

Comment:
Maybe explain why one would set this to true or false. What are the consequences?